### PR TITLE
Add reading of LM75 and RPI CPU temp to RPI CPU Interface

### PR DESCRIPTION
**Pull request by spiderkeys**
Reads the temperature from the LM75 attached to the RPI's I2C bus, as well as the RPI's internal CPU temperature, and emits the data on the cockpit bus to be logged by blackbox.js.

### CODE REVIEW

File: src/system-plugins/platform-manager/platforms/raspberrypi/boards/rev10/setup.js [hunk 1]
```diff
@@ -22,6 +32,26 @@ var SetupBoardInterface = function(board)
...
+        {
+          // First byte is temperature in C, first bit of second byte is equal to 0.5C
+          let temp = result[ 0 ] + ( ( ( result[ 1 ] & 0x80 ) >> 7 ) * 0.5 );
+
+          // Emit on cockpit bus
+          board.cockpit.emit( "board.temp.lm75", temp );
+        })
+        .catch( (err) =>
+        {
+          console.log( "Error reading LM75. Stopping task: " + err.message );
```
Comment (nerdgilbert) [inline src/system-plugins/platform-manager/platforms/raspberrypi/boards/rev10/setup.js]: Should be an error message?

**Comment by nerdgilbert** [comment]
Added a comment. Have you tested this with hardware? LGTM

File: src/system-plugins/platform-manager/platforms/raspberrypi/boards/rev10/setup.js (hunk 1 above)
Comment (spiderkeys) [inline src/system-plugins/platform-manager/platforms/raspberrypi/boards/rev10/setup.js]: Yep!

**Comment by spiderkeys** [comment]
Have tested the CPU temp on a rev8, but it doesn't have an LM75. I've tested the code used in the LM75 additions in a separate script on a Rev10 that does have one, and that worked, so I'm pretty confident this should be fine. Worst that will happen if there is an error in the code is the promise will throw and the periodic function will get canceled.

**Pull request merged**